src/abbfreeathome/freeathome.py

```diff
@@ -188,6 +191,15 @@ def _get_function_to_device_mapping(self) -> list[dict[str, Function | Base]]:
                 "function": Function.FID_WINDOW_DOOR_SENSOR,
                 "device_class": WindowDoorSensor,
             },
+            {
+                "function": Function.FID_DES_DOOR_RINGING_SENSOR,
+                "device_class": DesDoorRingingSensor,
+            },
+            {"function": Function.FID_SMOKE_DETECTOR, "device_class": SmokeDetector},
```

Comment:
You can keep the formatting by adding a trailing comma. Purely cosmetic purposes.

```suggestion
            {
                "function": Function.FID_SMOKE_DETECTOR,
                "device_class": SmokeDetector,
            },
```